controller_manager/include/controller_manager/controller_manager.h

```diff
@@ -178,7 +183,11 @@ class ControllerManager{
   std::vector<controller_interface::ControllerBase*> start_request_, stop_request_;
   std::list<hardware_interface::ControllerInfo> switch_start_list_, switch_stop_list_;
   bool please_switch_;
+  bool switch_started_;
   int switch_strictness_;
+  bool start_asap_;
+  double timeout_;
```

Comment:
See above at `switchController` for the name of this

Reply:
Fixed with the below refactor of the switch parameters?

Reply:
:+1: